v4/source/introduction.yaml

```diff
@@ -214,23 +214,184 @@ tags:
 
       The list of Mattermost WebSocket events are:
       - added_to_team
+        ```
+        {
+          'event': 'added_to_team',
+          'seq': <seq_id>,
+          'data': {
+            'team_id': '<team_id>',
+            'user_id': '<user_id>'
+          },
+          'broadcast': {
+            'omit_users': null,
+            'user_id': '<user_id>',
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>'
+          }
+        }
+        ```
       - authentication_challenge
-      - channel_converted
       - channel_created
+        ```
+        {
+          'event': 'channel_created',
+          'seq'  : <seq_id>,
+          'data' : {
+            'channel_id': '<channel_id>',
+            'team_id'   : '<team_id>'
+          },
+          'broadcast': {
+            'omit_users': null,
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>',
+            'user_id': '<user_id>'
+          }
+        }
+        ```
       - channel_deleted
-      - channel_member_updated
+        ```
+        {
+          'event': 'channel_deleted',
+          'seq': <seq_id>,
+          'data': {
+            'channel_id': '<channel_id>'
+           },
+          'broadcast': {
+            'omit_users': null,
+            'user_id': '<user_id>',
+            'channel_id': '<channel_id>',
+            'team_id': '<team_id>'
+          },
+
+          ```
       - channel_updated
+         ```
+         {
+           'event': 'channel_updated',
+           'seq': <seq_id>,
+           'data': {
+             'channel': {
+               'id': '<channel_id>',
+               'create_at': '<timestamp>',
+               'update_at': '<timestamp>',
+               'delete_at': '<timestamp>',
+               'team_id': '<team_id>',
+               'type': 'O',
+               'display_name':'<channel_id>',
+               'name': '<channel_name>',
+               'header': 'header',
+               'purpose': '',
+               'last_post_at': '<timestamp>',
+               'total_msg_count': '<total_msg_count>',
```

Comment:
There shouldn't be quotes here as this is a number. Same with the timestamp fields

Reply:
There's a lot of other cases throughout the changes with this issue